go.mod

```diff
@@ -223,4 +223,4 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 )
 
-replace github.com/go-git/go-git/v5 => github.com/argoproj-labs/go-git/v5 v5.4.4
+replace github.com/go-git/go-git/v5 => github.com/rohankmr414/go-git/v5 v5.4.7
```

Comment:
will update this once https://github.com/argoproj-labs/go-git/pull/6 is merged and released